front-end/src/app/app.component.ts

```diff
@@ -1,10 +1,43 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
+class Task {
```

Comment:
ezt az osztályt ki is emelhetnéd egy külön fájlba, és máshova beimportálnád. Belemehetne pl egy `src\model` könyvtárba. Így más osztály be tudná importálni, és elkülönülne a komponens kód a model osztályoktól.